### PR TITLE
Make editorial changes to the PRW exporter README

### PR DESCRIPTION
**Pull request by rakyll**

### CODE REVIEW

File: exporter/prometheusremotewriteexporter/README.md [hunk 1]
```diff
@@ -45,7 +48,7 @@ Example:
 ```yaml
 exporters:
   prometheusremotewrite:
-    endpoint: "http://some.url:9411/api/prom/push"
+    endpoint: "https://my-cortex:7900/api/v1/push"
```
Comment (rakyll) [inline exporter/prometheusremotewriteexporter/README.md:51]: @tigrannajaryan @bogdandrutu The example here is failing the link check. How do you normally deal with example links?

Comment (tigrannajaryan) [reply exporter/prometheusremotewriteexporter/README.md:51]: The linker checker has a (not very nice) way to disable checking a particular link: https://github.com/gaurav-nelson/github-action-markdown-link-check#disable-check-for-some-links

I wish there was a better way (maybe there is?).

Comment (rakyll) [reply exporter/prometheusremotewriteexporter/README.md]: Thanks for the pointers! Reverted this change back not to deal with it.

**Review by alolita — APPROVED** [review]
lgtm

**Review by carlosalberto — APPROVED** [review]

**Review by bogdandrutu — APPROVED** [review]

**Comment by bogdandrutu** [comment]
@rakyll seems that check-links does not like the Thanos link

**Comment by bogdandrutu** [comment]
`https://my-cortex:7900/api/v1/push` this one is the problem

**Comment by rakyll** [comment]
@bogdandrutu removed it, please take another look.

**Review by bogdandrutu — APPROVED** [review]

**Review by dashpole — APPROVED** [review]

**Pull request merged**